SharpGen.Runtime/ComActivationHelpers.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace SharpGen.Runtime
+{
+    public static class ComActivationHelpers
```

Comment:
All of the COM-specific types should live in SharpGen.Runtime.COM instead of in the SharpGen.Runtime base layer.

Reply:
I know, and yet I propose moving this class and the `CLSCTX` enum to `SharpGen.Runtime`, since it is independent of all other COM types, but provides bare minimum of the COM utilities. It is also used in tests in my `mine` branch. `IUnknown` and the rest will remain in `SharpGen.Runtime.COM`, along with the existing  `ComUtilities` class, which will be reimplemented on top of this class.